website/authentification/views.py

```diff
@@ -20,3 +20,20 @@ def signup(request):
 def logout_custom(request):
     logout(request)
     return redirect('home')
+
+def login_custom(request):
+    form = forms.LoginForm()
```

Comment:
```suggestion
    form = forms.LoginForm()
    if request.user.is_authenticated:
        return redirect('home')
```
Si l'utilisateur est déjà connecté, on redirige directement vers le home